install/docker/opensuse-docker-install.md

```diff
@@ -0,0 +1,71 @@
+# Install Docker for Kata Containers on openSUSE Leap
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../opensuse-installation-guide.md).
+>
+> - If you do not want to copy or type all these instructions by hand, you can use the
+>   [`kata-manager`](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager/kata-manager.sh)
+>   script to install the packaged system including your chosen container
+>   manager. Alternatively, you can generate a runnable shell script from
+>   individual documents using the
+>   [`kata-doc-to-script`](https://github.com/kata-containers/tests/blob/master/.ci/kata-doc-to-script.sh) script.
+
+1. Install the latest version of Docker with the following commands:
+
+   > **Note:** This step is only required if Docker is not installed on the system.
+
+   ```bash
+   $ sudo zypper -n install libcgroup1
+   $ sudo zypper -n install docker
+   ```
+
+   For more information on installing Docker please refer to the
+   [Docker Guide](https://software.opensuse.org/package/docker).
+
+2. Configure Docker to use Kata Containers with one of the following methods:
```

Comment:
For consistency with the other guides, please change this to the following (which includes "by default"):

> 2. Configure Docker to use Kata Containers by default with one of the following methods: